javascript/javascript1/week1/homework-week1-hossein.js

```diff
@@ -0,0 +1,47 @@
+// Javascript warm up part one
+let myFreeCodeCampUserName="@Hossein1065";
+
+// Age-ify (A future age calculator)
+var yearOfBirth =1986;
+var yearFuture=2025;
+var age=yearFuture-yearOfBirth;
```

Comment:
Using var is not the best practice, as in the near future it will be less reliable to use var especially in functions and loops. you can read more about it online "Let vs. var"
But rather using Let instead is always a safe choice, as long as you need to change its value later on.

```suggestion
let yearOfBirth =1986;
let yearFuture=2025;
let age=yearFuture-yearOfBirth;
```